marklogic-data-hub-central/src/main/java/com/marklogic/hub/central/cloud/aws/AWSConfiguration.java

```diff
@@ -17,63 +17,54 @@
 package com.marklogic.hub.central.cloud.aws;
 
 import com.amazonaws.services.s3.AmazonS3ClientBuilder;
-import com.amazonaws.services.simplesystemsmanagement.AWSSimpleSystemsManagementClientBuilder;
-import com.amazonaws.services.simplesystemsmanagement.model.GetParameterRequest;
 import com.marklogic.hub.central.web.SslUtil;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
 import org.springframework.boot.autoconfigure.web.ServerProperties;
 import org.springframework.boot.web.embedded.tomcat.TomcatServletWebServerFactory;
 import org.springframework.boot.web.server.WebServerFactoryCustomizer;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Primary;
 import org.springframework.context.annotation.Profile;
 
 import java.io.InputStream;
 
+import static com.marklogic.hub.central.cloud.Constants.CERTIFICATE_KEY_STORE_PASSWORD;
+
 @Configuration
 @Profile("aws")
 public class AWSConfiguration {
 
     private static final Logger logger = LoggerFactory.getLogger(AWSConfiguration.class);
 
-    @Value("${aws.s3.bucketName}")
+    @Autowired
+    AWSParameterStore awsParameterStore;
+
+    @Value("${aws.s3.bucketName:}")
     private String bucketName;
 
-    @Value("${aws.s3.keyName}")
+    @Value("${aws.s3.keyName:}")
     private String keyName;
 
-    private String keyStorePassword;
-
     @Bean
+    @Primary
```

Comment:
Needed to make this primary since Spring 2.3.8 was bringing in a new ServerProperties bean.